target/linux/realtek/Makefile

```diff
@@ -4,30 +4,18 @@ include $(TOPDIR)/rules.mk
 
 ARCH:=mips
 BOARD:=realtek
-BOARDNAME:=Realtek MIPS
-DEVICE_TYPE:=basic
+BOARDNAME:=Realtek based boards
 FEATURES:=ramdisk squashfs
 SUBTARGETS:=rtl838x rtl839x rtl930x rtl931x
 
 KERNEL_PATCHVER:=5.15
 
 define Target/Description
-	Build firmware images for Realtek RTL83xx based boards.
+	Build firmware images for Realtek RTL83xx/RTL93XX based boards.
```

Comment:
How about using a more generic name for all of the boards? If I add rtd129x, rtd1319 and rtd1619b, the list will be a little bit lengthy ? How about put these SoC specific in the subtarget makefile ?

Reply:
The name would be extended to RTL83xx/RTL93xx/RTD1xxx so not too bad. We should put the names somewhere, so users can find it easily. Oth, the subtargets will already list it anyway, so removing it is not a bad idea.